src/PdfReader/index.tsx

```diff
@@ -406,7 +420,14 @@ export default function usePdfReader(args: ReaderArguments): ReaderReturn {
     // do nothing if the reader is inactive
     if (state.state !== 'ACTIVE') return;
 
-    if (state.pageNumber > 1 && !state.settings.isScrolling) {
+    let startPage = 1;
+    if (manifest?.readingOrder && manifest?.readingOrder[state.resourceIndex]) {
```

Comment:
this is a preference, but I prefer 
`const startPage = manifest?.readingOrder && manifest?.readingOrder[state.resourceIndex] ? getStartPage(
        manifest?.readingOrder[state.resourceIndex].href
      ); : 1` as this makes the startPage variable immutable.  
      
      If it looks messy, I'd instead assign manifest?.readingOrder[state.resourceIndex] to a const and do the conditional on that.